conjur/logic/logout_logic.py

```diff
@@ -6,19 +6,22 @@
 This module is the business logic for logging out of the Conjur CLI
 """
 # pylint: disable=too-few-public-methods
+from conjur.interface.credentials_store_interface import CredentialsStoreInterface
+
+
 class LogoutLogic:
     """
     LogoutLogic
 
     This class holds the business logic for logging out of Conjur
     """
-    credentials = None
 
-    def __init__(self, credentials):
-        self.credentials = credentials
+    def __init__(self, credentials_provider : CredentialsStoreInterface):
+        self.credentials_provider = credentials_provider
+        self.number = 1
```

Comment:
I think this should be removed 😉

Reply:
![great_catch](https://user-images.githubusercontent.com/76062747/113558934-326fec80-9609-11eb-8710-8b9ffc80c5b4.gif)

great catch